content/rest/authentication.textile

```diff
@@ -130,26 +134,38 @@ Below is a rather contrived yet straightforward example that instances a REST li
   let rest = ARTRest(options: options)
 ```
 
+h3(#token-types). Token Types
+
+There are a couple of Ably-compatible tokens that you can have your server return to a client:
+
+h4(#ably-token). Ably Token
+
+An Ably Token is obtained through a "request":#request-token to Ably. This can either be done by the server, which then passes on the resulting Ably Token to the respective client, or by the client making use of an "Ably TokenRequest":#request-token. This token can also be embedded into an "External JWT":https://jwt.io as the @x-ably-token@ claim in the JWT's payload.
+
+h4(#ably-jwt). Ably JWT (JSON Web Token)
+
+"JWT (JSON Web Token)":https://jwt.io/ is an open standard (RFC 7519) that allows for compact secure information as JSON objects. Ably supports a JWT when the JWT is signed with an Ably API key. This type of JWT is called an "Ably JWT":/general/authentication#ably-jwt. A server can generate a JWT, and pass it to a client to be used. Unlike an Ably TokenRequest, it can be used as an actual token by the client, and the server can generate this token without the Ably library. Any third party software can be used to generate this JWT using the Ably private API key to sign it.
```

Comment:
"it can be used as an actual token by the client" -> "it can be used directly as a token by the client"